core/sql/executor/ExStats.cpp

```diff
@@ -3678,9 +3292,8 @@ void ExPartitionAccessStats::getVariableStatsInfo(char * dataBuffer,
 	      "AnsiName: %s PhysName: %s BuffersSize: %u BuffersSent: %u BuffersRcvd: %u NumMessages: %Ld MsgBytes: %Ld StatsBytes: %Ld MsgBytesSent: %Ld MsgBytesRcvd: %Ld ",
 	      ansiName_, fileName_,
 	      bufferStats()->sendBufferSize(), bufferStats()->sentBuffers().entryCnt(), bufferStats()->recdBuffers().entryCnt(), 
-	      fsDp2MsgsStats()->getNumMessages(),
-	      fsDp2MsgsStats()->getMessageBytes(),
-	      fsDp2MsgsStats()->getStatsBytes(),
+	      exeSEStats()->getNumIOCalls(),
+	      exeSEStats()->getNumIOBytes(),
```

Comment:
Thanks Arvind. I have fixed the issue with ExPartitionAccess::getVariableStatsInfo to provide 0 for statsBytes.  ExPartitionAccess is not used currently. Hence, the regression tests didn't detect this issue. Thanks for catching it. I have left ExPartitionAccess behind because it might be resurrected later